hmda/src/main/scala/hmda/persistence/submission/HmdaValidationError.scala

```diff
@@ -197,13 +197,21 @@ object HmdaValidationError
       case VerifyQuality(submissionId, verified, replyTo) =>
         if (List(Quality.code, QualityErrors.code, Macro.code, MacroErrors.code)
               .contains(state.statusCode)) {
-          Effect.persist(QualityVerified(submissionId, verified)).thenRun { _ =>
-            if (state.macroVerified) {
-              val updatedStatus = Verified
-              updateSubmissionStatus(sharding, submissionId, updatedStatus, log)
-              replyTo ! QualityVerified(submissionId, verified)
+          Effect
+            .persist(
+              QualityVerified(submissionId,
+                              verified,
+                              SubmissionStatus.valueOf(state.statusCode)))
+            .thenRun { _ =>
+              if (state.macroVerified) {
+                val updatedStatus = Verified
+                updateSubmissionStatus(sharding,
+                                       submissionId,
+                                       updatedStatus,
+                                       log)
+                replyTo ! QualityVerified(submissionId, verified, updatedStatus)
```

Comment:
Just want to make sure this behavior is correct.  This won't wait for the submission status to be updated, it fires off both requests (to update the submission status and the reply that the status is updated) simultaneously.  I'm completely fine with this solution, as it returns the expected result for the POST request.  But it is slightly different behavior than V1, when we waited for the status to be updated before sending a response.

Reply:
Trade-offs....I thought about it and decided that the code was going to get complicated, and we're doing this all over the place anyway (updating the status at the same time we reply). I'm OK with creating a ticket to suggest a change here, but this is probably fine for now.

Reply:
Sounds good!  It's definitely a trade-off and I agree with the design decision here to keep the code a bit simpler.